### ISSUE TITLE
Special Decks fail to load

### ISSUE THREAD

**Issue by BenMatteson** [brine/mtg-octgn#127]
among others, none of the decks in the special decks folder load. might be good to do a pass on the old ones to update/remove broken ones

**Comment by brine** [comment]
yeah I need to redo most of those decks due to the GUID changes.  Dunno how to approach it yet.

The planes and schemes should work again though.  Turns out they were just missing altogether.
